.github/workflows/ci.yml

```diff
@@ -1,13 +1,29 @@
-name: Lint
-
+name: CI
 on: [push]
 
 jobs:
-  eslint:
+  js-lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - uses: stefanoeb/eslint-action@1.0.2
-
-      - name: js-unit-tests
+  js-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: yarn install
+      - name: Test
         run: yarn test
+  py-markdown-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        uses: docker://docker
+        with:
+          args: docker build ./test/markdown/ --tag=markdown-tests:latest
+      - name: Test
+        uses: docker://docker
+        with:
+          args: docker run markdown-tests:latest
```

Comment:
So we need to pass the volume of files to this test. See the README.md file in test/markdown. I suppose I should add a simple check that at least one file is processed, because if there are no files, it passes but doesn't process any of them.

Reply:
Update: I added this test, merge via https://github.com/defund12/defund12.org/commit/2f06e4863ba8c3e5980a51fb46daad111b35f731

pull in the recent changes and you should see that it fails.